lib/identity_config.rb

```diff
@@ -59,6 +59,7 @@ def self.store
     config.add(:async_wait_timeout_seconds, type: :integer)
     config.add(:attribute_encryption_key, type: :string)
     config.add(:attribute_encryption_key_queue, type: :json)
+    config.add(:authentication_tmx_policy, type: :string, allow_nil: true)
```

Comment:
We set `allow_nil` here, but it's never `nil` based on the defaults. It might make sense for the default value to be `nil` though?